scripts/typescript-compatibility-build.yml

```diff
@@ -171,21 +171,21 @@ jobs:
           $compatibleTsVersions = "";
           $compatMapArray = Get-Content $env:BUILD_STAGINGDIRECTORY/compatMapDrop/compatMap.txt;
           foreach($line in $compatMapArray) {
-            if($i -match 0) {
-              $releaseVersion = $line;
-            } else {
-              if($i -match 1) {
-                $compatibleTsVersions += $line;
-              } else {
-                $compatibleTsVersions += ", ";
-                $compatibleTsVersions += $line;
-              }
-            }
-            $i++;
+          if($i -match 0) {
```

Comment:
why change this file? can you revert?

Reply:
seems to have tweaked the formatting for some reason